python/taulabs/readbyargs.py

```diff
@@ -0,0 +1,3 @@
+import telemetry
```

Comment:
So, when you import a python module, the code runs.

This creates a shorthand when you do from taulabs.readbyargs import uavo_list ... that it runs that method.  It's slight abuse of the module system but seemed like a convenient way to do this.